om4labs/diags/stress_curl/stress_curl.py

```diff
@@ -0,0 +1,259 @@
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import numpy as np
+import cmocean
+import xcompare
+import xarray as xr
+from xgcm import Grid
+
+from xcompare import plot_three_panel
+from om4labs.om4common import image_handler
+from om4labs.om4common import open_intake_catalog
+from om4labs.om4common import date_range
+from om4labs.om4parser import default_diag_parser
+
+
+def parse(cliargs=None, template=False):
+    """Function to capture the user-specified command line options
+
+    Parameters
+    ----------
+    cliargs : argparse, optional
+        Command line options from argparse, by default None
+    template : bool, optional
+        Return dictionary instead of parser, by default False
+
+    Returns
+    -------
+        parsed command line arguments
+
+    example usage:
+    om4labs stress_curl -s fname_static.nc --ref_taux omip_taux.nc  --ref_tauy omip_tauy.nc fname_model_taux.nc fname_model_tauy.nc
+    """
+
+    description = " "
+
+    exclude = ["basin", "hgrid", "topog", "gridspec", "config"]
+
+    parser = default_diag_parser(
+        description=description, template=template, exclude=exclude
+    )
+
+    parser.add_argument(
```

Comment:
I have a dilemma on what the best strategy for comparison is here. By doing the comparison inside om4labs, we need to duplicate read and calculate code (ref and model) before we can pass to xcompare and we always need a reference dataset. I think we could write the diag for one dataset and then I would build a generic wrapper to call 2 instances of the diag's read and calculate and pass the result to xcompare.

Reply:
after discussing with @jkrasting we still think the compare on top of om4labs is a good idea but will need some more work and I don't want to hold back this PR